### ISSUE TITLE
Neonatal infections

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6178]
**Preferred term label:**
Neonatal infections

**Synonyms**
Newborn infections

**Definition (free text, please give PubMed ID)**
Different infection syndromes during the neonatal period.
PMID: 27491687 

**Parent term (use hpo.jax.org/app)**
Unusual infection

**Comment by pnrobinson** [comment]
@johannaest  Sorry for the late reply. We generally try to avoid having <Onset> x <Phenotype> combinations in the HPO except if there is something different or special about the phenotype in the <Onset> period. The preferred way of doing this would be to use an onset term (in this case, https://hpo.jax.org/app/browse/term/HP:0003623) together with the phenotypic feature (https://hpo.jax.org/app/browse/term/HP:0002719).
